pymc_experimental/tests/test_model_builder.py

```diff
@@ -0,0 +1,128 @@
+import pymc as pm
+from pymc.model_builder import ModelBuilder
+import unittest
+import pytest
+import re
+from nose.tools import *
```

Comment:
I think nose is oldschool and only pytest should be used. Are you using any functionality of nose?